tools/ts_library_builder/ast_util.ts

```diff
@@ -0,0 +1,327 @@
+import { relative } from "path";
+import { readFileSync } from "fs";
+import { EOL } from "os";
+import {
+  EmitResult,
+  ExportDeclaration,
+  ImportDeclaration,
+  InterfaceDeclaration,
+  JSDoc,
+  Project,
+  PropertySignature,
+  SourceFile,
+  StatementedNode,
+  TypeGuards,
+  VariableStatement,
+  VariableDeclarationKind
+} from "ts-simple-ast";
+import * as ts from "typescript";
```

Comment:
@kitsonk sorry, didn't take a look at this until now. I recommend using the `ts` named export from "ts-simple-ast" instead. That way you ensure the code always uses the typescript dependency that ts-simple-ast is relying on (just in case they ever happen to be different).